README.md

```diff
@@ -8,7 +8,7 @@ LabVIEW 2017
 
 ## Dependencies
 
-- NI-SLSC >= 19.0
+- NI-SLSC >= 19.5
 - [Custom Device Message Library](https://github.com/ni/niveristand-custom-device-message-library)
```

Comment:
Why bump? I thought our baseline was always going to be SLSC 19.0. We should encourage everyone to use the latest but the minimum would still be 19.0 I think.

Reply:
We have to have SLSC Switch 19.1 in order to use the features we're adding. SLSC 19.5 is the driver that includes SLSC Switch 19.1.

Reply:
I'm well aware. That wasn't my point. Technically SLSC 19.0 is the absolute minimum driver that this custom device supports but only switches will work (not FIUs) and they'll need to use the legacy format for switches info.

After thinking about it some more, it probably better to list 19.5 in the readme as opposed to the true driver minimum, so users don't fall in the trap of only having switches support.

Reply:
> I'm well aware. That wasn't my point. Technically SLSC 19.0 is the absolute minimum driver that this custom device supports but only switches will work (not FIUs) and they'll need to use the legacy format for switches info.

Even if this is momentarily true due to how bare bones the UI is, we intend to make use of features added in SLSC Switch 19.1 in the very near future, and already do in the examples. E.g. `Get Topologies` and simulated sessions.

Reply:
Agreed. Let's keep NI-SLSC 19.5 as the minimum.

Reply:
Also, 191. is the minimum because the Switching library utilizes the Simulated feature that Ryan added in 19.1 Without SLSC 19.1, that library will be broken, so your custom device will not run. Use of that feature is already in, so 19.0 cannot be used.